### ISSUE TITLE
Feature Request: Option to configure Raspple II as WiFi bridge

### ISSUE THREAD

**Issue by oliverschmidt** [RasppleII/rasppleii#4]
In a few days there a new ADTPro with an additional Ethernet Virtual Drive client will become available. This new client will make use of the TCP/IP stack built into the 'Uthernet II' card. Doing so reduces the memory footprint of that client such that it can replace the Disk II driver inside ProDOS 8 just like the known serial client does.

So the Uthenet II becomes a viable alternative to one of the two serial connections necessary to take full advantage of A2CLOUD. And with the proposed web front-end for vsd (https://github.com/RasppleII/a2cloud/issues/31) the Uthernet II can even replace the second serial connection for the disk-mount-scenario.

So far so good. But Ethernet is replaced more and more in today's homes by WiFi. With the RPi 3 it's even built-in. This leaves the Ethernet connection of the RPi unused so it seems really natural to connect the Uthernet II directly to the RPi.

In such a setup a dual use is desirable:
- Access the ADTPro server (and the web front-end for vsd) on the RPi
- Use the RPi as WiFi bridge to access other machines / the Internet

A real layer-2-bridge of a WPA-secured WiFi client to Ethernet isn't possible so the usual approach is a so-called layer-3-pseudo-bridge. Fortunately Debian comes with pretty nice support for that :-)

In https://wiki.debian.org/BridgeNetworkConnectionsProxyArp the section "Automating the Process" describes the setup in detail. The great news is that this description applies to both Wheezy and Jessie 1:1 without any modification or addition :-))

I tried it on both and got the desired connectivity for the Uthernet II without further tweaking.

To summarize: Replacing one or even both serial connections from the A2 to the RPi with an Ethernet connection becomes now a from my perspective very appealing option. Especially when the RPi additionally brings WiFi to the existing TCP/IP application on the A2.

Therefore I ask you to consider an option to configure Raspple II as WiFi bridge as described in the Debian Wiki.
